js/viewer/annotation/base.js

```diff
@@ -60,36 +48,26 @@ function Base(attrs){
         var rst = "<" + tag + " ";
         var attr;
 
-        // if(this.parent.id !== ""){
-        //     rst += 'id="' + this.parent.id + '" ';
-        // };
-
-        var style = this.getStyle();
-        if (style.length > 0) {
-            this._attrs["style"] = style;
-        }
-
         for(attr in this._attrs){
-            // Ingoring style because that will be added at the end.
-            // Ignoreing vector-effect becasue this properpty is not needed in the output file
-            if (!this._attrs[attr] || this._attrs[attr] === null || attr === "style" || attr == "vector-effect"){
+            if (!this._attrs[attr] || this._attrs[attr] === null){
                 continue;
             }
 
             switch(attr){
```

Comment:
You should figure out what `tag` and `graph-id` are. I don't think we need them as attributes. Also you shouldn't just add `vector-effect` and instead should go over the `addedProperties` list. So if there's an attribute in `_attrs` that is part of `addedProperties`, we shouldn't add it to export and instead use the value that comes from `addedProperties`.

Reply:
`tag` = {rect, circle, line}
I still have to figure out why `graph-id` is used. From what I understand it is used for delete and highlight functionality.
I have added the check for `_addedProperties`. https://github.com/informatics-isi-edu/openseadragon-viewer/blob/4c4eb19b816508fafca08b8782afab2cf4796e76/js/viewer/annotation/base.js#L62